main.js

```diff
@@ -13,3 +24,9 @@ addEventListener('keydown', function (event) {
     $carImg.className = 'img-container left';
   }
 });
+
+addEventListener('keydown', function (event) {
+  if (event.key == ' ') {
+    var idInterval = setInterval(start, 16);
+  }
+});
```

Comment:
You should just add this to the already existing event listener for `keydown` above